components/MainContent.js

```diff
@@ -1,16 +1,17 @@
 import React from "react"
-
-function MainContent(props) {
+import data from "/data"
 
 //utility function to retreive data 
-    function getPageContent(selectedPage) {
-        console.log(data)
-        return data[selectedPage];
-    }
+function getPageContent(selectedPage) {
+    return data[selectedPage];
+}
+
+function MainContent(props) {
+ const data = getPageContent(props.selectedPage)
 
     return(
         <main className="main-content">
-            <h1>Some stuff</h1>
+            <h1>{data}</h1>
```

Comment:
Super close! Everything else looks good but it might have thrown an error since ‘data’ is an object. Do you see an error in the chrome console?

Reply:
i don't see any error

Reply:
Ah I wonder if it's because the data import was undefined without the `.` before the `/`

Reply:
it seems adding the `.` is breaking it. Odd!

Reply:
Oops, I didn't notice the`data` file was up a directory. So, we should talk about relative paths. 

When thinking about the current location of the file we are working in, when we import something from a directory up, we use `../` (double dots). Sort of like when navigating in the terminal and we use `cd ..` to go up a directory, the double dots mean the same thing here. 

So, when I mentioned using a single dot, it was because I thought the data directory was a sibling/neighbor to the file we were working in. If you use a double dot in stead, it will work as expected. (I just confirmed locally).

We'll need to talk a bit about the shape of the data we are importing. We can do that on screen share.